package.json

```diff
@@ -173,7 +173,7 @@
     "@stylelint/prettier-config": "^3.0.0",
     "@stylelint/remark-preset": "^5.1.1",
     "@types/balanced-match": "^1.0.4",
-    "@types/css-tree": "^2.3.8",
+    "@types/css-tree": "^2.3.9",
```

Comment:
After https://github.com/stylelint/stylelint/pull/8129/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R176, it's not necessary anymore.